e2e/testcases/cli_test.go

```diff
@@ -1278,6 +1278,37 @@ func TestNomosStatusNameFilter(t *testing.T) {
 	}
 }
 
+func TestVetSkipGVK(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.NomosCLI,
+		ntopts.SyncWithGitSource(nomostest.DefaultRootSyncID, ntopts.Unstructured),
+	)
+
+	rootRepo := nt.SyncSourceGitReadWriteRepository(nomostest.DefaultRootSyncID)
+
+	if err := rootRepo.Copy("../testdata/gatekeeper-skip-gvk", "acme"); err != nil {
```

Comment:
You can use `nt.Must()` here to simplify the code 
```suggestion
	nt.Must(rootRepo.Copy("../testdata/gatekeeper-skip-gvk", "acme"))
```